startup_assistance/indico_startup_assistance/operations.py

```diff
@@ -0,0 +1,24 @@
+# This file is part of the CERN Indico plugins.
+# Copyright (C) 2014 - 2018 CERN
+#
+# The CERN Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License; see
+# the LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from indico.core.db import db
+
+from indico_startup_assistance.models.startup_assistance_requests import StartupAssistanceRequest
+
+
+def create_startup_assistance_request(reservation):
+    startup_request = StartupAssistanceRequest()
```

Comment:
reservation.whatever = StartupAssistanceRequest()

and i think this whole thing is now so small that it can just go directly in wherever you call these functiosn here

Reply:
Yeah, yeah. Just like I mentioned. Initially I did it in a separate method because I didn't know how big the function will be. I will definitely move it back to `plugin.py` if necessary.